tests/test_nonbonded.py

```diff
@@ -129,7 +129,7 @@ def test_correctness(self):
         Test against the reference jax platform for correctness.
         """
         # we can't go bigger than this due to memory limitations in the the reference platform.
-        for N in [33, 65, 231, 1050, 4080]:
+        for N in [33, 65, 231, 1050, 3080]:
```

Comment:
The `jax.grad` ended up causing OOMs in the test. Best I can tell is that we were just scrapping by and with the addition of more tests it pushed it beyond the limit (https://stackoverflow.com/questions/20900380/py-test-excessive-memory-usage-with-large-number-of-tests).